src/hnt_contract.erl

```diff
@@ -78,6 +87,10 @@ get_hnt_balance(Account) ->
 update_from_l2(From, Nonce, NewPower, Burns) ->
     gen_server:call(?MODULE, {update, From, Nonce, NewPower, Burns}).
 
+%% @doc API for SubDAO contracts to burn HNT to get equivalent L2-DC
+burn_into_l2(From, Nonce, Pubkey, Amt) ->
```

Comment:
Why is there a nonce, and there's no "destination" to burn into, it's not always the same as the account doing the burn.

Reply:
Updated in 7fd8b24